marathon-interface/src/main/scala/mesosphere/marathon/interface/auth/Identity.scala

```diff
@@ -0,0 +1,14 @@
+package mesosphere.marathon.interface.auth
+
+/**
+  * Base trait that represents an identity.
+  */
+trait Identity {
+
+  /**
+    * The string identifier of that identity.
+    * @return the unique identifier of that user.
+    */
+  def id: String
```

Comment:
At the moment this is a guess and should probably be replaced:
- we need to log which identity performs which action (or we do it on the plugin side)
- we will need to pass the user identity down to mesos